extensions/packageManager/src/pages/Library.tsx

```diff
@@ -724,22 +724,68 @@ const Library: React.FC = () => {
                     {/* display "v1.0 installed" if installed, or "install v1.1" if not" */}
                     {isInstalled(selectedItem) && selectedVersion === installedVersion(selectedItem) ? (
                       <span>
-                        {selectedVersion} {strings.installed}
+                        <span
+                          css={{
+                            maxWidth: 80,
+                            textOverflow: 'ellipsis',
+                            overflow: 'hidden',
+                            whiteSpace: 'nowrap',
+                            display: 'inline-block',
+                          }}
+                          title={selectedVersion}
+                        >
+                          {selectedVersion}
+                        </span>{' '}
```

Comment:
Why this blank space here? Can we use padding or margin to get this extra space

Reply:
^ you can also use &nbsp;

Reply:
you know, the prettier plugin inserted those! Yes, I will use an nbsp.

Reply:
done